test/parallel/test-cluster-shared-leak.js

```diff
@@ -15,6 +15,11 @@ if (cluster.isMaster) {
   worker1 = cluster.fork();
   worker1.on('message', common.mustCall(function() {
     worker2 = cluster.fork();
+    worker2.on('error', function(e) {
+      // EPIPE is OK on Windows
+      if ((! common.isWindows) || (e.code !== 'EPIPE'))
```

Comment:
I was going to write it that way but I went with this to more closely mirror the (simpler) condition involving `ECONNRESET` later in the file. You're right, though, and I'll switch it.